### PR TITLE
Updated Releases Notes Draft

### PR DESCRIPTION
**Pull request by dstrebel**
Draft from working document

### CODE REVIEW

File: releases/release-1.14/release-notes-draft.md [hunk 1]
```diff
@@ -1,54 +1,113 @@
+## 1.14 What’s New
+
+
+Support for Windows Nodes is Going GA (#116)
```
Comment (marpaia) [inline releases/release-1.14/release-notes-draft.md]: Unfortunately all of these have to be [#116](https://github.com/kubernetes/enhancements/issues/116) like:

```
[#116](https://github.com/kubernetes/enhancements/issues/116)
```

Or for k/k:

```
[#1234](https://github.com/kubernetes/kubernetes/issues/1234)
```

Comment (onyiny-ang) [reply releases/release-1.14/release-notes-draft.md]: I noticed that and will correct it. Thanks @marpaia

**Comment by spiffxp** [comment]
/close
old draft at this point

**Pull request closed without merging**